server/core/src/main/java/dev/slimevr/tracking/processor/skeleton/SpineYawCorrection.kt

```diff
@@ -0,0 +1,76 @@
+package dev.slimevr.tracking.processor.skeleton
+
+import com.jme3.math.FastMath
+import dev.slimevr.VRServer
+import dev.slimevr.config.YawCorrectionConfig
+import dev.slimevr.tracking.trackers.Tracker
+import io.github.axisangles.ktmath.EulerOrder
+import io.github.axisangles.ktmath.Quaternion
+import io.github.axisangles.ktmath.Vector3.Companion.POS_Y
+import kotlin.math.*
+
+class SpineYawCorrection(
+	val skeleton: HumanSkeleton,
+	private val yawCorrectionConfig: YawCorrectionConfig,
+) {
+
+	private val upperBodyTrackers = filterImuTrackers(
+		skeleton.headTracker,
+		skeleton.neckTracker,
+		skeleton.upperChestTracker,
+		skeleton.chestTracker,
+		skeleton.waistTracker,
+		skeleton.hipTracker,
+	)
```

Comment:
This needs to be updated when the trackers are updated from `HumanSkeleton.setTrackersFromList()`

Reply:
Also does it need to be a list? It feels like an array would suffice still, though it matters less with this usage.

Reply:
Nice catch! I kept it as a list so that I can use `zipWithNext`.

Addressed by changing `SpineYawCorrection` into a stateful class with `setTrackers` and `setConfig`.